packages/xod-client-electron/src/app/arduinoActions.js

```diff
@@ -134,9 +134,13 @@ export const checkPort = port => listPorts()
  * @param {PatchPath} patchPath Entry-point patch path
  * @returns {Promise<String, Error>} Promise with transpiled code or Error
  */
-export const doTranspileForArduino = ({ project, patchPath }) =>
+export const doTranspileForArduino = ({ project, patchPath, options = {} }) =>
```

Comment:
Monsterification detected. Instead of the current

```js
export const doTranspileForArduino = ({ project, patchPath, options = {} }) => { ... }
```

could it be:

```js
// simpler name, no param destruction
export const transpileForArduino = (project, patchPath, options = {}) => { ... }
```